services/movie_session.py

```diff
@@ -0,0 +1,48 @@
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=cinema_hall_id,
```

Comment:
To associate the MovieSession with a CinemaHall, consider passing cinema_hall_id as a parameter)